projects/sunbird-pdf-player/src/lib/sunbird-pdf-player.service.ts

```diff
@@ -22,6 +22,14 @@ export class SunbirdPdfPlayerService {
     this.config = config;
     this.playSessionId = this.utilService.uniqueId();
 
+    if (!context?.channel){
```

Comment:
Where are the default configs mentioned in documentation set here

Reply:
Default config added if channel and pdata is not available in context